x-pack/plugins/cases/public/components/system_actions/cases/cases_params.tsx

```diff
@@ -39,11 +39,20 @@ export const CasesParamsFieldsComponent: React.FunctionComponent<
   ActionParamsProps<CasesActionParams>
 > = ({ actionParams, editAction, errors, index, producerId, featureId }) => {
   const {
+    cloud,
```

Comment:
Just a reminder that the `cases_params` component is rendered through various applications that provide the `KibanaContextProvider`. Even if the triggers actions UI provides `cloud` maybe other solutions will not. If it is working in Security then they definitely provide it 🙂.

Reply:
But this portion of cases' code is only used by `triggers_actions_ui` so it's safe to assume that cloud will be provided no? I added cloud to triggers actions UI because they define Kibana context for connectors.

> If it is working in Security then they definitely provide it 🙂. 
The important part is that it works in Stack Management so that is independent of security right? (even if the project is security serverless)

Reply:
This portion of code is being used everywhere the alerts flyout is being used. Also, the actions component is being exported and used by the Security solution in their rules form:

<img width="2559" alt="Screenshot 2024-10-11 at 2 13 16 PM" src="https://github.com/user-attachments/assets/b9ccbb1e-7284-4b5e-a296-45a932dcde0c">

Nevertheless, even if the `cloud` plugin is not available because you can only create security rules there the `owner` will always be correct. The issue is on Stack so it should be fine.